From b10b0d43851adeddde5b1e97a1e5ae6658a2c43e Mon Sep 17 00:00:00 2001
From: Alexandru Gagniuc <mr.nuke.me@gmail.com>
Date: Sun, 18 Sep 2022 21:21:14 -0500
Subject: [PATCH 1011/1012] ath11k: Disable coldboot calibration for IPQ6018

As robimarko noted for IPQ8074:

"
  There is a bug with the remoteproc reset after coldboot calibration,
  so until that is resolved disabled it to allow using the radio.
 "
---
 drivers/net/wireless/ath/ath11k/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath11k/core.c b/drivers/net/wireless/ath/ath11k/core.c
index 48a449fbd2bc..4a7d71d7aeed 100644
--- a/drivers/net/wireless/ath/ath11k/core.c
+++ b/drivers/net/wireless/ath/ath11k/core.c
@@ -109,7 +109,7 @@ static const struct ath11k_hw_params ath11k_hw_params[] = {
 		.supports_monitor = true,
 		.supports_shadow_regs = false,
 		.idle_ps = false,
-		.cold_boot_calib = true,
+		.cold_boot_calib = false,
 		.supports_suspend = false,
 		.hal_desc_sz = sizeof(struct hal_rx_desc_ipq8074),
 		.fix_l1ss = true,
-- 
2.37.1

